### PR TITLE
Design feedback

### PR DESCRIPTION
**Pull request by nkukadiya89**

### CODE REVIEW

File: src/PrivateRoute.js [hunk 1]
```diff
@@ -0,0 +1,31 @@
+import * as React from "react";
+import { Route, Redirect } from "react-router-dom";
+import AdminLayout from "./layouts/AdminLayout";
+
+const PrivateRoute = ({ component: Component, ...rest }) => {
```
Comment (dtrodger) [inline src/PrivateRoute.js:5]: Also cool :)

File: src/components/employees/EmployeeDetail.js [hunk 2]
```diff
@@ -0,0 +1,146 @@
...
+import {
+    Form,
+    InputGroup,
+    Row,
+    Col,
+    Button,
+    Table,
+    Modal,
+  } from "react-bootstrap";
+const EmployeeDetail = (props) => {
```
Comment (dtrodger) [inline src/components/employees/EmployeeDetail.js:11]: New line after imports

File: src/components/employees/NewEmployee.js [hunk 3]
```diff
@@ -0,0 +1,103 @@
+import React from "react";
+import {
+    Form,
+    InputGroup,
+    Row,
+    Col,
+    Button,
+    FormControl,
+  } from "react-bootstrap";
+const NewEmployee = (props) => {
```
Comment (dtrodger) [inline src/components/employees/NewEmployee.js:10]: New line

File: src/components/equipment/AddEditInventoryCategory.js [hunk 4]
```diff
@@ -0,0 +1,87 @@
+import React from "react";
+import { Form, InputGroup, Row, Col, Button } from "react-bootstrap";
+import { ArrowRight, Search } from "react-bootstrap-icons";
+const AddEditInventoryCategory = (props) => {
```
Comment (dtrodger) [inline src/components/equipment/AddEditInventoryCategory.js:4]: I'll stop commenting this, but new line after imports

File: src/components/equipment/Bags.js [hunk 5]
```diff
@@ -0,0 +1,103 @@
+import React, { useState } from 'react';
+import { Table,  Form, Button, InputGroup,Modal,Row, Col} from "react-bootstrap";
+import {ArrowRight, Search } from "react-bootstrap-icons"
+import { Link } from "react-router-dom";
+import BagDetails from "./BagDetails";
+import NewBag from "./NewBag";
+  const Bags = (props) => {
```
Comment (dtrodger) [inline src/components/equipment/Bags.js:7]: Indent back

File: src/components/equipment/EquipmentJobs.js [hunk 6]
```diff
@@ -0,0 +1,161 @@
+import React, { useState } from "react";
+import { Table, Form, Button, Modal } from "react-bootstrap";
+import JobDetails from "./JobDetails";
+import FullCalendar from "@fullcalendar/react";
```
Comment (dtrodger) [inline src/components/equipment/EquipmentJobs.js:4]: Kyle will integrate the React big calendar component here

File: src/components/equipment/Inventory.js [hunk 7]
```diff
@@ -0,0 +1,173 @@
+import { Table, Form, Button, InputGroup, Modal,Row, Col } from "react-bootstrap";
+import React, { useState } from "react";
+import { ArrowRight, Search } from "react-bootstrap-icons";
+import { Link } from "react-router-dom";
+import AddEditInventoryCategory from "./AddEditInventoryCategory";
+import NewInventoryItem from "./NewInventoryItem";
+import InventoryDetails from "./InventoryDetails";
+
+const Inventory = (props) => {
+  // const onBtnNewInventoryItem = () =>{
```
Comment (dtrodger) [inline src/components/equipment/Inventory.js:10]: Remove comments

File: src/components/equipment/InventoryEdit.js [hunk 8]
```diff
@@ -0,0 +1,162 @@
...
+            <FormControl size="lg" autoFocus type="text" value="B&H" />
+          </Form.Group>
+        </Col>
+          </Row>
+          <Row>
+          
+        </Row>
+      
+        
+
```
Comment (dtrodger) [inline src/components/equipment/InventoryEdit.js:96]: Clean up commented out code and new lines like this before PRs

File: src/components/equipment/JobDetails.js [hunk 9]
```diff
@@ -0,0 +1,285 @@
...
+        <Col xs={12} md={12}>
+          <Form.Group>Make sure students are wearing proper attire</Form.Group>
+        </Col>
+      </Row>
+      <div className="heading_one">Equipment</div>
+      <Table hover responsive className="mb-3 table_grid white_table">
+        <thead>
+          <tr>
+            <th>Employee</th>
+            <th style={{ minWidth: "140px" }}>Bags/Equipment </th>
```
Comment (dtrodger) [inline src/components/equipment/JobDetails.js:168]: Remove inline styling

File: src/components/equipment/NewBag.js [hunk 10]
```diff
@@ -0,0 +1,121 @@
...
+              <Col xs={12} md={12}>
+                <Form.Group>
+                  <Button variant="link"><strong>Assign Bag to Employee</strong> </Button>
+                </Form.Group>
+              </Col>
+            </Row>
+          
+           
+            {/* <Row>
+            <Col xs={12} md={3}>
```
Comment (dtrodger) [inline src/components/equipment/NewBag.js:92]: No commented out code in PRs

File: src/components/equipment/Repairs.js [hunk 11]
```diff
@@ -0,0 +1,149 @@
...
+import NewRepair from "./NewRepair";
+import RepairEdit from "./RepairEdit";
+
+const Repairs = (props) => {
+  const [newrepair, newrepairShow] = useState(false);
+  const [editRepairState, setEditRepairState] = useState(false);
+
+  return (
+    <>
+      <div className="search_head justify-content-end">
```
Comment (dtrodger) [inline src/components/equipment/Repairs.js:13]: Use consistent naming convention for class names. No - use _

File: src/components/equipment/Repairs.js [hunk 12]
```diff
@@ -0,0 +1,149 @@
...
+        <Button variant="primary" size="sm" onClick={() => newrepairShow(true)}>
+          New Repair
+        </Button>
+      </div>
+      <Table hover responsive>
+        <thead>
+          <tr>
+            <th>Date Sent Out</th>
+            <th>Date Received</th>
+            <th style={{ minWidth: "140px" }}>Name </th>
```
Comment (dtrodger) [inline src/components/equipment/Repairs.js:23]: Remove inline css

File: src/containers/Login.js [hunk 13]
```diff
@@ -22,61 +22,80 @@ export default function Login() {
...
-    }).then(function(result){
-      const session = result.data.tokenAuth;
-      setSession(session);
-      localStorage.setItem("session", JSON.stringify(session));
-      userHasAuthenticated(true);
-    }).catch(error => {
-      onError(error);
-      setIsLoading(false);
-    })
+    props.history.push('/equipment');
```
Comment (dtrodger) [inline src/containers/Login.js:25]: Cant merge a PR without auth

File: src/layouts/AdminLayout.js [hunk 14]
```diff
@@ -0,0 +1,161 @@
...
+  Button,
+  DropdownButton,
+  Dropdown
+} from "react-bootstrap";
+import { LinkContainer } from "react-router-bootstrap";
+import { Link } from "react-router-dom";
+import { PersonCircle, Search } from "react-bootstrap-icons";
+const AdminLayout = (props) => {
+  const handleLogout = () => {
+    //localStorage.removeItem("session");
```
Comment (dtrodger) [inline src/layouts/AdminLayout.js:19]: Cant merge a PR without working auth

File: src/layouts/AdminLayout.js [hunk 15]
```diff
@@ -0,0 +1,161 @@
...
+                  </NavDropdown.Item>
+                  <NavDropdown.Item className="text-bump">
+                    Clothing Tips
+                  </NavDropdown.Item>
+                </NavDropdown>
+              </Nav.Item>
+
+              <Nav.Item>
+                <NavDropdown title="+ New" id="nav-dropdown">
+                  {/* <NavDropdown.Item
```
Comment (dtrodger) [inline src/layouts/AdminLayout.js:71]: Remove commented out code

File: src/components/equipment/Repairs.js (hunk 11 above)
Comment (nkukadiya89) [reply src/components/equipment/Repairs.js:13]: "justify-content-end" class is bootstrap class. So now our custom class name replaced with '-' (search-page) for bootstrap class and our custom class consistency.

File: src/layouts/AdminLayout.js (hunk 14 above)
Comment (nkukadiya89) [reply src/layouts/AdminLayout.js]: In logout code comment, i am not getting what auth-related changes need to do?  As per my understanding, i need to remove logged in user detail from the local storage session. so i am enabled this code. please share your feedback on this comment

**Pull request merged**